cli/command/service/opts.go

```diff
@@ -182,7 +182,7 @@ func (o *SecretOpt) Set(value string) error {
 
 		value := parts[1]
 		switch key {
-		case "source":
+		case "source", "src":
```

Comment:
Looking at the UX for `docker secret ls`;

```bash
$ docker secret ls
ID                          NAME                    CREATED                                   UPDATED
mhv17xfe3gh6xc4rij5orpfds   secret.json             2016-10-27 23:25:43.909181089 +0000 UTC   2016-10-27 23:25:43.909181089 +0000 UTC
```

As a user I would expect `name` or `id` to be options for this, possibly `alias` for the name inside the container

/me continues painting bike shed